src/main/java/org/mockito/verification/VerificationEvent.java

```diff
@@ -0,0 +1,31 @@
+package org.mockito.verification;
+
+import org.mockito.internal.verification.api.VerificationData;
+
+/**
+ * Contains all information about a verification that has happened.
+ */
+public interface VerificationEvent {
```

Comment:
Add an `@Incubating` there as well. Since it may opportunistically change with `Optional` once the team switches to mockito 3 which will be based on JDK8.